modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceDiagrams.path

```diff
@@ -37,6 +37,11 @@
 	<td></td>
 </tr>
 <!--DIAGRAM_MAPPING-->
+<tr>
+	<td>DIAGRAM_FILE</td>
+	<td>//div[contains(@class,'image-wrapper cropper')]/img[contains(@src,'key_image')]</td>
```

Comment:
key_image needs to be set as a variable like '${key_image}' based off of how you have written your test. Otherwise this path is always searching for something specific.